### ISSUE TITLE
Save dungeons played by subjects 

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#414]
Information to reproduce each played dungeon should be saved is each subject directory. Remember that we will need to algorithmically load and measure the novelty of each dungeon after the study.

**Comment by jakeryang** [comment]
Added graph .DOT files and the dungeon they played to the Subject's dir.

**Comment by schrum2** [comment]
Please write some code of otherwise provide some kind of example test that definitively shows that these saved levels can be loaded and played, and otherwise analyze, and confirm that the level matches what was originally played.

**Comment by jakeryang** [comment]
There is already code that saves the dungeon and loads it back up. This is how I did the retries. When the game started I would save the dungeon and then let the player play. When they wanted to retry, I loaded the dungeon from the JSON. When they exit it deletes the JSON. When you play the rouge-like (when you press enter), look in the `data/rouge/tmp` folder and there's a dungeon.json which is a copy of the dungeon before you played.
